layouts/default.vue

```diff
@@ -4,22 +4,18 @@
     <v-content class="pa-0">
       <v-card tile height="100%" width="100%">
         <toolbar v-model="drawer" :is-index="isIndex" @log="handleLog" />
-        <v-alert :value="$nuxt.isOffline" type="error" dismissible class="ma-0">
-          <!-- eslint-disable-next-line -->
-          You are currently offline. Please check your internet connection.
-        </v-alert>
         <v-container :fill-height="isIndex" fluid class="pa-0 mx-0">
           <nuxt />
         </v-container>
         <v-snackbar
-          v-model="snackbar"
+          v-model="$nuxt.isOffline"
```

Comment:
This removes the previous trigger which lets the user know if they can't use geolocation. You will have to add the `$nuxt.isOffline` to the computed property as was previously done.